src/hooks/useSingleCampaignHandler.ts

```diff
@@ -7,13 +7,14 @@ import {
   getCampaigns,
   getSmartContractWithProvider,
 } from "../utils/ContractUtils";
-import { useProvider } from "wagmi";
+import { useAccount, useProvider } from "wagmi";
 import toast from "react-hot-toast";
 import DaoSeederFactory from "@daoseeder/core/artifacts/contracts/DaoSeederFactory.sol/DaoSeederFactory.json";
 import { ICampaign } from "../interfaces/ICampaign";
 
 export const useSingleCampaignHandler = () => {
   const { state } = useLocation();
+  const { address } = useAccount();
```

Comment:
this should be renamed to userAddress because it's being exported from here

Reply:
Done